### PR TITLE
Make Travis Green Again

### PR DESCRIPTION
**Pull request by pp3345**

### CODE REVIEW

**Comment by pp3345** [comment]
Ignore the bot. Just added a space between `#` and `TODO`. I don't have a clue what's going on with pylint in Travis now though.

**Comment by mithro** [comment]
Looks like pylint is having trouble importing the code;
```
40.51s$ pylint --reports=n *.py edid_parser/ frontend/
Using config file /home/travis/build/timvideos/edid.tv/pylintrc
************* Module edid.tv.edid_parser.tests
E:  6, 0: No name 'EDIDParser' in module 'edid_parser' (no-name-in-module)
E:  6, 0: No name 'EDIDParsingError' in module 'edid_parser' (no-name-in-module)
E:  6, 0: No name 'DisplayType' in module 'edid_parser' (no-name-in-module)
E:  6, 0: No name 'DisplayStereoMode' in module 'edid_parser' (no-name-in-module)
E:  6, 0: No name 'TimingSyncScheme' in module 'edid_parser' (no-name-in-module)
E:  6, 0: No name 'CVTSupportDefinitionPreferredAspectRatio' in module 'edid_parser' (no-name-in-module)
E:  6, 0: No name 'ColorBitDepth' in module 'edid_parser' (no-name-in-module)
E:  6, 0: No name 'DigitalVideoInterface' in module 'edid_parser' (no-name-in-module)
************* Module edid.tv.frontend.selenium_tests.test_update
E:  8, 0: Unable to import 'base' (import-error)
************* Module edid.tv.frontend.selenium_tests.test_login
E:  7, 0: Unable to import 'base' (import-error)
************* Module edid.tv.frontend.selenium_tests.test_upload
E:  8, 0: Unable to import 'base' (import-error)
-----------------------------------
Your code has been rated at 9.82/10
```

Might try `PYTHONPATH=$PYTHONPATH:.` in front of the command? See https://stackoverflow.com/questions/1899436/pylint-unable-to-import-error-how-to-set-pythonpath ?

**Comment by mithro** [comment]
> Alternatively, edit ~/.pylintrc to include the directory above your module, like this:
> ```ini
> [MASTER]
> init-hook='import sys; sys.path.append("/path/to/root")'
> ```
> (Or in other version of pylint, the init-hook requires you to change [General] to [MASTER])
> 
> Both of these options ought to work.
> 
> Hope that helps.

**Comment by pp3345** [comment]
Neither of these seem to work. I don't really know what is causing it to fail now in the first place, since it didn't have any issues finding stuff previously. Also, it's only failing for _some_ imports.

**Comment by pp3345** [comment]
It seems to work this way, however, now it doesn't include all subfolders of `frontend/`. Adding each single folder to the command line seems a bit hacky.

**Comment by mithro** [comment]
Great work!

File: .travis.yml [hunk 1]
```diff
@@ -10,9 +10,9 @@ install:
   - pip install coveralls
 # command to run tests
 script:
-  - pep8 --statistics *.py
-  - pep8 --statistics edid_parser/
-  - pep8 --statistics frontend/
+  - pycodestyle --statistics --exclude=wsgi.py *.py
+  - pycodestyle --statistics edid_parser/
+  - pycodestyle --statistics frontend/
   - pylint --reports=n *.py edid_parser/*.py frontend/*.py
```
Comment (mithro) [inline .travis.yml]:
```
pylint --reports=n $(find -name \*.py)
```

**Pull request merged**